.yarnrc

```diff
@@ -1,5 +1,5 @@
 disturl "https://electronjs.org/headers"
-target "27.2.3"
-ms_build_id "26495564"
+target "28.1.4"
```

Comment:
28.2.1 was released yesterday.

Reply:
It will be updated when internal builds are available.